docker-compose.yml

```diff
@@ -27,5 +27,7 @@ services:
     image: postgres
     environment:
       - POSTGRES_USER=mds
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
For context, this is the error caused on a first run when this isn't provided.

```
db_1         | Error: Database is uninitialized and superuser password is not specified.
db_1         |        You must specify POSTGRES_PASSWORD to a non-empty value for the
db_1         |        superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
db_1         |
db_1         |        You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
db_1         |        connections without a password. This is *not* recommended.
db_1         |
db_1         |        See PostgreSQL documentation about "trust":
db_1         |        https://www.postgresql.org/docs/current/auth-trust.html
metadata-service_db_1 exited with code 1
```



The error kills the db so migrations and the app fail. It looks like the current intent is passwordless auth so I added `POSTGRES_HOST_AUTH_METHOD=trust` to continue that strategy and get things working on a fresh `up`. The `Error` turns to a `Warning` then and works properly.
```